File: Backend/src/main/java/com/feedle/feedleapi/Networking/DeleteUserRequest.java
package com.feedle.feedleapi.Networking;

public class DeleteUserRequest extends Request {
    private int userId;

    public DeleteUserRequest(int userId) {
        super(RequestType.DeleteUser);
        this.userId = userId;
    }

    public int getUserId() {
        return userId;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/DeleteUserRequest.cs
using System.Text.Json.Serialization;

namespace FeedleDataTier.Network
{
    public class DeleteUserRequest : Request
    {
        [JsonPropertyName("userId")]
        public int UserId { get; set;}

        public DeleteUserRequest(int userId) : base(RequestType.DeleteUser)
        {
            this.UserId = userId;
        }
    }
}